doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -532,6 +532,7 @@ Libraries for networking
 
     * Moved JSON manipulation from :file:`nrf_cloud_fota.c` to :file:`nrf_cloud_codec_internal.c`.
     * Fixed a build issue that occurred when MQTT and P-GPS are enabled and A-GPS is disabled.
+    * Fixed bug preventing `AIR_QUAL` from being enabled in shadow UI service info.
```

Comment:
```suggestion

  * Fixed:

    * A build issue that occurred when MQTT and P-GPS are enabled and A-GPS is disabled.
    * A bug preventing `AIR_QUAL` from being enabled in shadow UI service info.
```
"Fixed" can be used as its own bullet on the same level as "updated" in the changelog.